ground/src/main/proto/com/google/ground/job.proto

```diff
@@ -19,11 +19,189 @@ syntax = "proto3";
 option java_package = "com.google.android.ground.proto";
 option java_multiple_files = true;
 
-import "com/google/ground/style.proto";
-
+// A single job, defined by one or more tasks, and zero or more locations of
+// interest (aka data collection sites).
 message Job {
+    // UUID of the job.
     string id = 1;
-    string name = 2;
-    Style default_style = 3;
-    // TODO(#1748): Add `tasks` and `strategy` fields.
+    // Index of the job when shown in lists.
+    int32 index = 2;
+    // User-facing name of the job.
+    string name = 3;
+    // The style used when rendering geometries and other UI elements associated
+    // with this job.
+    Style default_style = 4;
```

Comment:
nit: Should this just be `style` if there is no way to set a style to override it?

Reply:
If we apply YAGNI aggressively then yes! I assumed we'd need dynamic styling at some point, but since we don't have it already I can rename, eso, he new Firestore representation lets us change names easily!